aws/eks-customer/scripts/deploy-utility.sh

```diff
@@ -0,0 +1,132 @@
+}
+
+function deploy_utility() {
+  echo $utilities_json | jq -c '.[]' | while read -r utility; do
+		utility_name=$(echo "$utility" | jq -r '.name')
+    cluster_label_type=$(echo "$utility" | jq -r '.cluster_label_type')
+
+    add_utility_to_application_file $utility_name $cluster_label_type
+    replace_custom_values $utility_name
+    commit_changes "CLUSTER_NAME: ${CLUSTER_NAME} Adding utility ${utility_name}" $application_yaml
+    wait_for_healthy $utility_name
+
+	done
+
+}
+
+function add_utility_to_application_file() {
+	utility_name=$1
+  cluster_label_type=$2
+  echo "Adding utility ${utility_name} to application-values.yaml"
+
+  if [[ -f $application_yaml && $cluster_label_type != "customer" ]]; then
+      CLUSTER_NAME_exists=$(yq eval ".applications[] | select(.name == \"$utility_name\") | .cluster_labels | select(.[] | .\"cluster-id\" == \"${CLUSTER_NAME}\")" $application_yaml)
+      if [[ -z $CLUSTER_NAME_exists ]]; then
+          yq eval "(.applications[] | select(.name == \"$utility_name\") | .cluster_labels) += [{\"cluster-id\": \"${CLUSTER_NAME}\"}]" -i $application_yaml
+      fi
+  fi
+
+  stage_changes $application_yaml
+}
+
+function replace_custom_values () {
+  echo "Replacing custom values"
+	utility_name=$1
+
+  certificate_arn=$(escape_string ${CERTIFICATE_ARN})
+  private_certificate_arn=$(escape_string ${PRIVATE_CERTIFICATE_ARN})
+  allow_list_cidr_range=$(escape_string ${ALLOW_LIST_CIDR_RANGE})
+
+  if [[  -f $gitops_apps_dir/custom-values-template/$utility_name-custom-values.yaml-template ]]; then
+    cp $gitops_apps_dir/custom-values-template/$utility_name-custom-values.yaml-template $gitops_apps_dir/${ENV}/helm-values/${CLUSTER_NAME}/$utility_name-custom-values.yaml
+    sed -i "s/<CLUSTER_ID>/${CLUSTER_NAME}/g" $gitops_apps_dir/${ENV}/helm-values/${CLUSTER_NAME}/$utility_name-custom-values.yaml
+    sed -i "s/<ENV>/${ENV}/g" $gitops_apps_dir/${ENV}/helm-values/${CLUSTER_NAME}/$utility_name-custom-values.yaml
+    sed -i "s/<CLUSTER_ID>/${CLUSTER_NAME}/g" $gitops_apps_dir/${ENV}/helm-values/${CLUSTER_NAME}/$utility_name-custom-values.yaml
+    sed -i "s/<CERTFICATE_ARN>/$certificate_arn/g" $gitops_apps_dir/${ENV}/helm-values/${CLUSTER_NAME}/$utility_name-custom-values.yaml
+    sed -i "s/<PRIVATE_CERTIFICATE_ARN>/$private_certificate_arn/g" $gitops_apps_dir/${ENV}/helm-values/${CLUSTER_NAME}/$utility_name-custom-values.yaml
+    sed -i "s/<VPC_ID>/${VPC_ID}/g" $gitops_apps_dir/${ENV}/helm-values/${CLUSTER_NAME}/$utility_name-custom-values.yaml
+    sed -i "s/<PRIVATE_DOMAIN>/${PRIVATE_DOMAIN}/g" $gitops_apps_dir/${ENV}/helm-values/${CLUSTER_NAME}/$utility_name-custom-values.yaml
+    sed -i "s/<IP_RANGE>/$allow_list_cidr_range/g" $gitops_apps_dir/${ENV}/helm-values/${CLUSTER_NAME}/$utility_name-custom-values.yaml
+    sed -i "s/hostNetwork: false/hostNetwork: true/g" $gitops_apps_dir/${ENV}/helm-values/${CLUSTER_NAME}/$utility_name-custom-values.yaml
```

Comment:
maybe something
```
sed -i -e "s/<CLUSTER_ID>/${CLUSTER_NAME}/g" \
       -e "s/<ENV>/${ENV}/g" \
       -e "s/<CERTIFICATE_ARN>/$certificate_arn/g" \
       ... \
       "$target_file"
```
since the target file is the same